dev/ci/README.md

```diff
@@ -97,36 +107,29 @@ so that it doesn't, or provide a branch fixing these developments (or at
 least work with the author of the development / other Coq developers to
 prepare these fixes). Then, add an overlay in
 [`dev/ci/user-overlays`](/dev/ci/user-overlays) (see the README there)
-in a separate commit in your PR.
+as part of your PR.
 
 The process to merge your PR is then to submit PRs to the external
 development repositories, merge the latter first (if the fixes are
-backward-compatible), drop the overlay commit and merge the PR on Coq then.
+backward-compatible), and merge the PR on Coq then.
 
 See also [`test-suite/README.md`](/test-suite/README.md) for information about adding new tests to the test-suite.
 
 
-Travis specific information
----------------------------
-
-Travis rebuilds all of Coq's executables and stdlib for each job. Coq
-is built with `./configure -local`, then used for the job's test.
-
-
-GitLab specific information
----------------------------
+Advanced GitLab CI information
+------------------------------
 
-GitLab is set up to use the "build artifact" feature to avoid
+GitLab CI is set up to use the "build artifact" feature to avoid
 rebuilding Coq. In one job, Coq is built with `./configure -prefix
 install` and `make install` is run, then the `install` directory
```

Comment:
AFAICT the prefix here is `_install_ci`.

Reply:
Indeed.